packages/client/src/comp/AddressForm.tsx

```diff
@@ -124,18 +131,61 @@ export const RawAddressForm: React.FC<{rawState: string, zip?: string}> = ({rawS
         (<a target='_blank' rel='noopener noreferrer' href='https://www.vote.org/am-i-registered-to-vote/'>Unsure if you are registered?</a>)
       </p>
       <FlexBox>
-        <FlexGrow>
+        <FlexGrow style={{ flex: 1 }}>
```

Comment:
Let's make
- address (75%) / apartment (25%) of a line
- city (50%) / state (25%) / zipcode (25%) for tablet and above

But on mobile, the last line becomes:
- city (100%)
- state (50%) / zipcode (50%)